README.md

```diff
@@ -1 +1,52 @@
-# Hello-world4
\ No newline at end of file
+# Hello-world4
+
+> In this project, repository was set to master all of the tools and best practices that I learnt about(Documenting repo in a professional way,Git & GitHub & GitHub Flow and Linters) in previous steps.
+
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Deployment
+
```

Comment:
- [ ] Kindly make sure you give the instructions on how your repos were set up in the `setup`
- [ ] Kindly give the instruction on how to use this repos once it's set up locally in the `Usage`
- [ ] And delete the checkpoints that did not apply to this project.